functions/src/movieClubs/movies/movieFunctions.ts

```diff
@@ -210,5 +210,31 @@ exports.likeMovie = functions.https.onCall(
 exports.dislikeMovie = functions.https.onCall(
   async (request: CallableRequest<LikeMovieData>) => {
 
+    try {
+      const { data, auth } = request;
+      const { uid } = verifyAuth(auth);
+
+      const requiredFields = ["movieClubId", "movieId", "name"];
+      verifyRequiredFields(data, requiredFields);
+
+      await verifyMembership(uid, data.movieClubId);
+
+      const movieDocRef = getMovieDocRef(data.movieId, data.movieClubId)
+
+      let params = {
+        dislikedBy: firebaseAdmin.firestore.FieldValue.arrayUnion(data.name),
+        likedBy: firebaseAdmin.firestore.FieldValue.arrayRemove(data.name),
+      }
+
+      if (data.undo){
+        params.dislikedBy = firebaseAdmin.firestore.FieldValue.arrayRemove(data.name)
+      }
+
+      await movieDocRef.update(params);
+
+      return { success: true };
+    } catch (error) {
+      handleCatchHttpsError('Error liking comment:', error);
```

Comment:
This message is wrong

```suggestion
      handleCatchHttpsError('Error disliking comment:', error);
```